Resources/Prototypes/Entities/Objects/Specific/rehydrateable.yml

```
@@ -55,7 +39,52 @@
         - LowImpassable
 
 - type: entity
-  parent: MonkeyCube
+  parent: RehydratableItem
+  id: RehydratableAnimalCube
+  abstract: true
+  name: "rehydratable animal cube"
```

```suggestion
  name: rehydratable animal cube
```
You can also just remove this line completely since it is abstract.